stories/slider/slider.stories.js

```diff
@@ -0,0 +1,127 @@
+import '../../dist/slider.css';
+
+export default {
+    title: 'Components/Slider',
+    parameters: {
+        description: `A slider is a control that enables the user to adjust single values within a specified numerical range. A range slider is a user interface control that enables the user to select a value range within a predefined numerical
+interval.`
+    },
+    tags: ['f3', 'theme']
+};
+
+/**
+ * Basic slider
+ */
+
+export const basic = () => `
+<div style="width: 50%">
+    <div class="fd-slider" role="slider">
+        <div class="fd-slider__wrapper">
+            <div class="fd-slider__track">
+                <div class="fd-slider__track-range" style="width: 50%;"></div>
+                <div class="fd-slider__handle" style="left: 50%;"></div>
+            </div>
+        </div>
+    </div>
+</div>
+`;
+
+/**
+ * Range slider
+ */
+
+export const range = () => `
+<div style="width: 50%">
+    <div class="fd-slider" role="slider">
+        <div class="fd-slider__wrapper">
+            <div class="fd-slider__track">
+                <div class="fd-slider__track-range" style="width: 40%;left: 40%"></div>
+                <div class="fd-slider__handle" style="left: 40%;"></div>
+                <div class="fd-slider__handle" style="left: 80%;"></div>
```

Comment:
why do you add inline styling here? Our components should have inline styling only for better representation in the documentation (usually it's applied on the container div).

Reply:
I feel like the inline css might be ok _for this component only_. It's being used to display the different states of the slider without having to go through the trouble of setting up the interaction to work.

We should take the `<div style="width: 50%">` off the wrapper and use one of the docs classes instead though. You can find some containers here - https://github.com/SAP/fundamental-styles/blob/master/.storybook/custom/custom.scss or add another if none of those fit the needs of this component.

Reply:
@jbadan that's right, inline styling on slider handles was used to give them different positions so that way user sees a more or less real-world example of how range slider would look like. Of course, in component implementation these values are going to be set dynamically.

I'll use helper class from custom scss file to set wrapper's width, thanks